src/babel-serializer/network/pcap.cc

```diff
@@ -1,67 +1,90 @@
 #include "pcap.hh"
 
-babel::pcap::data babel::pcap::read(cc::span<const std::byte> data, read_config const& cfg, babel::error_handler on_error)
-{
-    babel::pcap::data r;
+#include <clean-core/utility.hh>
 
-    read(data, [&](auto const& packet, auto) { r.packets.push_back(packet); }, [&](auto const& header) { r.header = header; }, cfg, on_error);
+namespace
+{
+template <class T>
+T byteswap(T v)
+{
+    static_assert(std::is_pod_v<T>);
+    std::byte d[sizeof(v)];
+    cc::as_byte_span(v).copy_to(cc::as_byte_span(d));
+    for (size_t i = 0; i < (sizeof(v) / 2); ++i)
+        cc::swap(d[i], d[sizeof(v) - 1 - i]);
+    cc::as_byte_span(v).copy_from(cc::as_byte_span(d));
+    return v;
+}
+}
```

Comment:
can this be replaced with your new cc::byteswap?

Reply:
(or, is it ever required assuming the cc version is ADL-jacked)

Reply:
as T is in {uint32_t, uint64_t, etc.} it cannot be ADL jacked
but ofc, now that we have the swap in cc that one should be used.

Reply:
I replaced this, now that `cc::byteswap` is merged :+1: